src/seedu/addressbook/data/person/Name.java

```diff
@@ -43,6 +44,10 @@ public static boolean isValidName(String test) {
         return Arrays.asList(fullName.split("\\s+"));
     }
 
+    public int compareTo(Name otherName) {
+        return this.fullName.compareTo(otherName.fullName);
```

Comment:
Should compareTo be case-insensitive instead? Since we are comparing names.

Reply:
Now that I think about it, yes it should. Thank you for the feedback! I will update it to reflect this